concepticondata/conceptlists/Luangthongkum-2019-341.tsv

```diff
@@ -0,0 +1,342 @@
+Luangthongkum-2019-341-139	139	hawk, eagle	2683	EAGLE OR HAWK
+Luangthongkum-2019-341-140	140	have, be, exist, stay, there is		
+Luangthongkum-2019-341-141	141	head	1256	HEAD
+Luangthongkum-2019-341-142	142	heart	1223	HEART
+Luangthongkum-2019-341-143	143	heavy	1210	HEAVY
+Luangthongkum-2019-341-144	144	hemp	2041	HEMP
+Luangthongkum-2019-341-145	145	hip	1745	HIP
+Luangthongkum-2019-341-146	146	hit hard (with a fist)	353	STRIKE
+Luangthongkum-2019-341-147	147	horn	1393	HORN (ANATOMY)
+Luangthongkum-2019-341-148	148	hornet	3261	HORNET
+Luangthongkum-2019-341-149	149	horse	615	HORSE
+Luangthongkum-2019-341-150	150	hundred	1634	HUNDRED
+Luangthongkum-2019-341-151	151	husband	1200	HUSBAND
+Luangthongkum-2019-341-152	152	ill, painful, sick	2103	PAINFUL OR SICK
+Luangthongkum-2019-341-153	153	insect	620	INSECT
+Luangthongkum-2019-341-154	154	intestines	1991	INTESTINES
+Luangthongkum-2019-341-155	155	iron	621	IRON
+Luangthongkum-2019-341-156	156	itchy	149	ITCHY
+Luangthongkum-2019-341-157	157	kidney	489	KIDNEY
+Luangthongkum-2019-341-158	158	leaf	628	LEAF
+Luangthongkum-2019-341-159	159	leech (land~)	2273	LEECH
+Luangthongkum-2019-341-160	160	leech (water~)	949	WATER LEECH
+Luangthongkum-2019-341-161	161	left (side)	244	LEFT
+Luangthongkum-2019-341-162	162	leg	1297	LEG
+Luangthongkum-2019-341-163	163	lemur (flying~), flying squirrel		
+Luangthongkum-2019-341-164	164	lick (v.)	319	LICK
+Luangthongkum-2019-341-165	165	lie down, sleep	2116	REST OR SLEEP
+Luangthongkum-2019-341-166	166	light (~weight)	1052	LIGHT (WEIGHT)
+Luangthongkum-2019-341-167	167	linear measure of about 8-9 inches (a span)		
+Luangthongkum-2019-341-168	168	linear measure of about half a yard (closed-hand)		
+Luangthongkum-2019-341-169	169	linear measure of about half a yard (open-hand)		
+Luangthongkum-2019-341-170	170	linear measure of about two yards		
+Luangthongkum-2019-341-171	171	listen to	1608	LISTEN
+Luangthongkum-2019-341-172	172	live, stay		
+Luangthongkum-2019-341-173	173	liver	1224	LIVER
+Luangthongkum-2019-341-174	174	long (of time)	418	FOR A LONG TIME
+Luangthongkum-2019-341-175	175	long (of length), tall, high	2143	LONG OR TALL
+Luangthongkum-2019-341-176	176	look after (things)		
+Luangthongkum-2019-341-177	177	loris		
+Luangthongkum-2019-341-178	178	louse (chicken~), insect		
+Luangthongkum-2019-341-179	179	louse (head~)	310	HEAD LOUSE
+Luangthongkum-2019-341-180	180	lung	688	LUNG
+Luangthongkum-2019-341-181	181	maggot		
+Luangthongkum-2019-341-182	182	male (of animals), father		
+Luangthongkum-2019-341-183	183	man, male	1554	MAN
+Luangthongkum-2019-341-184	184	mango	2398	MANGO
+Luangthongkum-2019-341-185	185	many, much	1198	MANY
+Luangthongkum-2019-341-186	186	mat	195	MAT
+Luangthongkum-2019-341-187	187	milipede		
+Luangthongkum-2019-341-188	188	mole	1907	MOLE
+Luangthongkum-2019-341-189	189	money	1559	MONEY
+Luangthongkum-2019-341-190	190	monitor lizard (land~)	3017	MONITOR LIZARD
+Luangthongkum-2019-341-191	191	monitor lizard (water~)		
```

Comment:
MONITOR LIZARD includes both terrestrial and aquatic monitor lizards. But I can see the point in mapping it to the former as that seems to be the vast majority of monitor lizard species.

Reply:
I can see the point why they can be left unmapped but it seems like the term is used to refer mostly to the land species (also reflected in Snider-2004-1700-1104) while the water species are called specifically as such.

Reply:
I'd leave it as it is.